src/Symfony/Component/Yaml/Yaml.php

```diff
@@ -30,6 +30,7 @@ class Yaml
     const DUMP_MULTI_LINE_LITERAL_BLOCK = 128;
     const PARSE_CONSTANT = 256;
     const DUMP_EMPTY_ARRAY_AS_SEQUENCE = 1024;
+    const PARSE_KEYS_AS_STRING = 2048;
```

Comment:
If you just parse the YAML file and do not want to dump it afterwards, this may be enough.